lib/ava-files.js

```diff
@@ -205,7 +205,7 @@ class AvaFiles {
 		const initialPatterns = this.files.concat(excludePatterns);
 
 		// Like in `api.js`, tests must be `.js` files and not start with `_`
-		if (path.extname(filePath) !== '.js' || path.basename(filePath)[0] === '_') {
+		if (!this.extensions.includes(path.extname(filePath).substring(1)) || path.basename(filePath)[0] === '_') {
```

Comment:
Good point. I didn’t know ava supports compound extensions, since I didn’t introduce this usage of `path.extname` here. Anyway I can change this of course, the question is should I only change it in this line or should I replace all usages of `path.extname` in the ava codebase?

Reply:
I’ve adapted all usages in `ava-files`